public/docs/ts/_cache/guide/template-syntax.jade

```diff
@@ -858,8 +858,21 @@ block style-property-name-dart-diff
   header [()] = banana in a box
   :marked
     To remember that the parentheses go inside the brackets, visualize a *banana in a box*.
+
++ifDocsFor('ts|js')
+  .callout.is-important
+    header FormsModule is Required to use ngModel
+    :marked
+      Before we can use `ngModel` two-way data binding, we need to import the `FormsModule`
```

Comment:
Before we can use the `ngModel`  directive in a two-way data binding we must import the `FormsModule` .